0006/README.md

```diff
@@ -0,0 +1,31 @@
+---
+id: 0009
+title: Automated GitHub Action Runner
+status: ideation 
+authors: David McKay <david@rawkode.com>
+---
+
+## Summary:
+
+The `tink` and `portal` repositories use GitHub Actions to run their automated tests on pull-request and push to master.
+
+A dedicated GitHub Action runner is provided for the [Tinkerbell organization](https://github.com/tinkerbell), this runner was manually created and provisioned by [Gianluca](https://github.com/gianarb); this leaves the runner vulnerable; should it suffering any problems, restoring it will be a manual process.
+
+
+## Goals:
+
+- Automate the creation and provisioning of the dedicated runner
+- Convert all Tinkerbell repositories to using GitHub Actions with our dedicated runner
```

Comment:
This is a non-goal.
The runner only works for the tinkerbell/tink repository when a Vagrant CI check is required.